mods/d2k/rules/vehicles.yaml

```diff
@@ -337,6 +337,11 @@ devastator:
 		Step: 5
 		Ticks: 3
 		HealIfBelow: 50%
+	AttackDetonates:
```

Comment:
My understanding is that this should only be available if the unit is at critical health.